src/Util.cpp

```diff
@@ -72,6 +72,43 @@ bool FileExists(const char *filename)
 #endif
 }
 
+#ifdef _WIN32
+wchar_t* utf8ToUtf16(const char *utf8)
+{
+    wchar_t *utf16 = NULL;
```

Comment:
You can use the nicer C++11 version of `NULL` called `nullptr`.

Reply:
Done.